ranger/gui/ui.py

```diff
@@ -116,14 +116,17 @@ def initialize(self):
             if self.settings.update_tmux_title and 'TMUX' in os.environ:
                 try:
                     self._tmux_automatic_rename = check_output(
-                        ['tmux', 'show-window-options', '-v', 'automatic-rename']).strip()
+                        ['tmux', 'show-window-option', '-v', 'automatic-rename']).strip()
```

Comment:
that lef out `s` is on purpose. Command is actually `show-window-option`, `tmux show-window-options` produces no output.

Reply:
`man tmux` suggests the official command *is* "show-window-option**s**", both work for me but I prefer that one. It doesn't return a result for me unless the `-g` flag is passed though. I think you have to pass a target window otherwise. Though I get no output even if I provide a target, not even when I ask for all options rather than just `automatic-rename`.